runtime/runtime/src/state_viewer/mod.rs

```diff
@@ -152,6 +152,111 @@ impl TrieViewer {
         access_keys
     }
 
+    pub fn view_gas_key(
+        &self,
+        state_update: &TrieUpdate,
+        account_id: &AccountId,
+        public_key: &PublicKey,
+        include_nonces: bool,
+    ) -> Result<near_primitives::views::GasKeyView, errors::ViewGasKeyError> {
```

Comment:
Since we have `use near_primitives::views::{GasKeyView,` at the top, why we do we specify full path for GasKeyView here?